cmake/kconfig.cmake

```diff
@@ -5,6 +5,22 @@
 file(MAKE_DIRECTORY ${PROJECT_BINARY_DIR}/kconfig/include/generated)
 file(MAKE_DIRECTORY ${PROJECT_BINARY_DIR}/kconfig/include/config)
 
+# Support multiple SOC_ROOT
+set(WRITE WRITE)
```

Comment:
Kind of tricky when reading this for the first time... why not call this OPERATION or WRITE_OP or something?

Reply:
`OPERATION` is good, will use.

Reply:
> `OPERATION` is good, will use.

Still planning on doing this?

Reply:
done